### PR TITLE
Create config

### PR DESCRIPTION
**Pull request by LindaAlblas**

### CODE REVIEW

File: client/gulpfile.js [hunk 1]
```diff
@@ -89,9 +90,24 @@ gulp.task('sass:watch', function () {
     gulp.watch('assets/styles/sass/*.scss', ['compile-sass']);
 });
 
+gulp.task('create-config-dev', function (){
```
Comment (hunt3ri) [inline client/gulpfile.js]: Rename to create-dev-config

File: client/gulpfile.js [hunk 2]
```diff
@@ -89,9 +90,24 @@ gulp.task('sass:watch', function () {
...
+    /** Creates a config file for Angular with the relevant environment variables for development */
+   return gulp.src('taskingmanager.config.json')
+       .pipe(config('taskingmanager.config', {environment: 'development'}))
+       .pipe(gulp.dest('app'))
+});
+
+gulp.task('create-config-build', function (){
```
Comment (hunt3ri) [inline client/gulpfile.js]: Rename to create-release-config

File: client/taskingmanager.config.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "development": {
+    "EnvironmentConfig":{
+      "tmAPI": "/api"
+    }
+  },
+  "build": {
+    "EnvironmentConfig":{
```
Comment (hunt3ri) [inline client/taskingmanager.config.json]: Rename to release

**Comment by LindaAlblas** [comment]
Renamed tasks and variables

**Pull request merged**